olm-catalog/serverless-operator/project.yaml

```diff
@@ -55,10 +55,10 @@ dependencies:
     func:
         util: quay.io/boson/alpine-socat:1.7.4.3-r1-non-root
         tekton_s2i: registry.redhat.io/source-to-image/source-to-image-rhel8@sha256:6a6025914296a62fdf2092c3a40011bd9b966a6806b094d51eec5e1bd5026ef4
-        tekton_buildah: registry.redhat.io/rhel8/buildah@sha256:55f5c7d88c77e1ab5a2ab8e877172b2b66cb196e21f10dc45148470d0ee8bbae
-        nodejs_16: registry.access.redhat.com/ubi8/nodejs-16@sha256:720a3a2594a1804b5ecd7046a5ce4be82885bfa7c83df4c9bcade0e1e00db078
-        openjdk_17: registry.access.redhat.com/ubi8/openjdk-17@sha256:b00f687d913b8d1e027f7eabd6765de6c8d469629bef9550f10dbf207af24fe5
-        python-39: registry.access.redhat.com/ubi8/python-39@sha256:cf0af1732c483d4e6ba708f9f4d5541cb43c98c3c67c604c23b0e55897eebe41
+        tekton_buildah: registry.redhat.io/rhel8/buildah@sha256:b48f410efa0ff8ab0db6ead420a5d8d866d64af846fece5efb185230d7ecf591
+        nodejs_16: registry.access.redhat.com/ubi8/nodejs-16@sha256:f90b28179d1d5de68d42ef1a9166d70a9dfea85a1c6a4bf12dce2c59eeb2fbb1
```

Comment:
is this `sha` some info that needs to be looked up somewhere, than added here?
No way to automate? E.g. get sha of `latest` ?

Reply:
Yes, usually I checked latest image and find sha, may be we can think to automate (not sure why we add it here) or we can move this to the prod config file and replace placeholders downstream.

Reply:
Yes, I usually find out latest and copy SHA (manually), and we can think about the automation (not sure about the reason why we add it here) or we can move this prod config to update SHA at downstream.